.gitignore

```diff
@@ -3,3 +3,5 @@
 Homestead.yaml
 Homestead.json
 .env
+
```

Comment:
Normally, it's better to keep IDE-specific git ignore files in your global ignore:

https://davidwalsh.name/global-gitignore